### ISSUE TITLE
Invalid site ID

### ISSUE THREAD

**Issue by chrisrowe** [verbb/navigation#150]
### Description
Getting `Invalid site ID` exceptions on both the front and back end.

- Plugin version: 1.3.13
- Craft version: 3.5b2

**Comment by engram-design** [comment]
Can you provide a bit more information? Is it when editing a node, or just any page?

Can you post the full error with `devMode` on?

**Comment by chrisrowe** [comment]
Sorry I've downgraded now. When editing any menu in the backend, and when using `nodes` in the front end.
AFAICT there's nothing odd about my usage so you should be able to replicate it easily.

**Comment by engram-design** [comment]
I can't replicate it at all on a few different installs, so that's odd. What version did you downgrade to?

**Comment by engram-design** [comment]
Oh - sorry, i just noticed you're testing on Craft 3.5 beta 2 - I've not tested it on that, so that's probably it.

**Comment by chrisrowe** [comment]
3.5b1, let me try again

**Comment by chrisrowe** [comment]
Yeah, beta-2

**Comment by engram-design** [comment]
Should be fixed in [1.3.14](https://github.com/verbb/navigation/releases/tag/1.3.14)

**Comment by chrisrowe** [comment]
<3 Thanks Josh!
